resources/tests/GanacheTests/CombinedExample.obs

```diff
@@ -0,0 +1,17 @@
+main contract CombinedExample {
```

Comment:
Please make the indentation consistent.

Reply:
Also, the filename says this is an example, but it's not clear what it's an example of. I think the real answer is "everything to date," but that's not a good description in the long term. Can you choose a more explanatory name?

Reply:
I've renamed it (and the version with return) as "CombinedBasics" -- they are tests that combine all the basics of the language, sort of everything before the first interesting feature (which is structure types).